src/filesystems/qsys/QSysFs.ts

```diff
@@ -265,4 +279,34 @@ export class QSysFS implements vscode.FileSystemProvider {
     delete(uri: vscode.Uri, options: { readonly recursive: boolean; }): void | Thenable<void> {
         throw new FileSystemError("Method not implemented.");
     }
+}
+
+const SOURCE_DATES_RESET_WARNING = 'sourceDatesResetWarning';
+async function warnAboutSourceDates() {
+    const storage = instance.getStorage();
+    if (!storage?.hasMessageBeenShown(SOURCE_DATES_RESET_WARNING)) {
+        const save = l10n.t("Save");
+        const dismiss = l10n.t("Save & don't show again");
+        const abort = l10n.t("Abort");
+        const openSettings = l10n.t("Abort & open settings");
+        const choice = await vscode.window.showWarningMessage(l10n.t("Source dates support is disabled. Saving now will set all the dates to 0. Do you whish to proceed anyway?"),
```

Comment:
Spelling mistake:

```suggestion
        const choice = await vscode.window.showWarningMessage(l10n.t("Source date support is disabled. Saving now will set all the dates to 0. Do you wish to proceed anyway?"),
```

I also suggest a rewording of `source dates` - `date` should be singular when in context of `support`. Or as alternative, `Source dates are disabled` without `support`...

Reply:
Agreed, thanks!